Kernel/VM/RingBuffer.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
```

Comment:
Since you wrote the RingBuffer completely by yourself, how about copyrighting it to yourself? :)

Reply:
Alright, will do :)